themes/api.jquery.com/index.php

```diff
@@ -2,6 +2,10 @@
 <div class="content-right listing twelve columns">
 	<div id="content">
 		<h1 class="page-title">jQuery API</h1>
+		<p>jQuery is a fast, small, and feature-rich JavaScript library. It makes things like HTML document traversal and manipulation, event handling, animation, and Ajax much simpler with an easy-to-use API that works across a multitude of browsers. If you're new to jQuery, you might be interested in the <a href="http://learn.jquery.com/">jQuery Learning Center</a> tutorials.</p>
+
+		<p>This site provides API documentation for the latest verions of jQuery (1.11.2 and 2.1.3). If you're working with an older verion of jQuery, you can find its API docs in this sidebar. However, we would encourage you to upgrade to the newest version in order to receive the best support and take advantage of recent bug fixes and enhancements.</p>
```

Comment:
> This site provides API documentation for the latest verions of jQuery (1.11.2 and 2.1.3)

This isn't exactly true. This provides documentation for all versions, you just need to pay attention to the version info on each page. Anyway, APIs don't change for patch versions and keeping this up to date will drive @dmethvin nuts.